packages/botonic-plugin-contentful/tests/contentful/manage/manage-contentful-entry.test.ts

```diff
@@ -75,4 +80,44 @@ describe('ManageContentful entries', () => {
       sut.deleteContent(context, TEST_NEW_CONTENT_ID)
     ).rejects.toThrow(CmsException)
   })
+
+  test('TEST: convertValueType converts the ButtonStyle and the FollowUp', async () => {
```

Comment:
test name is not accurate. it should not refer to how it's implemented, but about the feature.
change to st like updateFields is able to manage fields of type ButtonStyle and FollowUps